src/main/java/com/parasoft/demoapp/grpc/service/JsonServiceImpl.java

```diff
@@ -81,4 +94,68 @@ public void getItemsInStock(StreamObserver<ItemEntity> responseObserver) {
             log.error(e.getMessage(), e);
         }
     }
+
+    @Override
+    public StreamObserver<ItemRequest> updateItemsInStock(StreamObserver<ItemResponse> responseObserver) {
+        return new StreamObserver<ItemRequest>() {
+            @Override
+            public void onNext(ItemRequest value) {
+                try {
+                    synchronized (clock) {
+                        ParameterValidator.requireNonNull(value, AssetMessages.REQUEST_PARAMETER_CANNOT_BE_EMPTY);
+                        ParameterValidator.requireNonNull(value.getValue(), AssetMessages.IN_STOCK_CANNOT_BE_NULL);
```

Comment:
Change the message to OPERATION_NUMBER_CANNOT_BE_NULL=The quantity for item inventory dedction or addition can not be null.